app/views/topics/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h2>Topics</h2>
+<p>
```

Comment:
You're using a `<p>` tag here, which is fine for text, but you're also nesting a table/list inside it. If you just want a container element, why not use a `<div>`, or, even better, something more semantic (https://www.w3schools.com/html/html5_semantic_elements.asp)